home/user/repos/blitzed-out/vite.config.js

```
@@ -0,0 +1,24 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import path from 'path';
+
+export default defineConfig({
+  plugins: [react()],
+  resolve: {
+    alias: {
+      '@': path.resolve(__dirname, './src'),
```

ESLint

[error] '__dirname' is not defined.

(no-undef)